src/containers/buttonscreen.js

```diff
@@ -0,0 +1,29 @@
+
+
+
+  const mapStateToProps = (state) => ({
```

Comment:
I'm guessing that this file is related to phase 2. Please keep this branch only related to phase 1 work.